src/main/java/com/github/stickerifier/stickerify/telegram/model/TelegramRequest.java

```diff
@@ -88,11 +88,12 @@ private String getUsername() {
 	}
 
 	public Answer getAnswerMessage() {
-		return isHelpCommand() ? HELP : ABOUT;
+		return isHelpOrStartCommand() ? HELP : ABOUT;
 	}
 
-	private boolean isHelpCommand() {
-		return HELP_COMMAND.equalsIgnoreCase(message.text());
+	private boolean isHelpOrStartCommand() {
+		return HELP_COMMAND.equalsIgnoreCase(message.text()) ||
```

Comment:
To be consistent with the rest of the project, you should put the binary operator at the beginning of the next line